REQUIRE

```diff
@@ -1,4 +1,4 @@
-julia 0.6
+julia 0.7-alpha
```

Comment:
I think `julia 0.7-` is enough (?)

Reply:
Nope, apparently not.
It will  not decide to make any changes with that